core/domain/draft_upgrade_services_test.py

```diff
@@ -236,10 +307,21 @@ def test_convert_states_v30_dict_to_v31_dict(self):
                         }
                     }
                 }
-            }).to_dict())
+            })]
+        # Migrate exploration to state schema version 31.
+        self.perform_and_assert_exploration_states_schema_migration('30', '31')
+        # Migrate the draft change list's state schema to the migrated
+        # exploration's schema.
+        migrated_draft_change_list_v31 = (
+            draft_upgrade_services.try_upgrading_draft_to_exp_version(
+                draft_change_list_v30, 1, 2, self.EXP_ID)
+        )
+        self.assertEqual(migrated_draft_change_list_v31[0].to_dict(),
+                         draft_change_list_v31[0].to_dict())
 
     def test_convert_states_v29_dict_to_v30_dict(self):
-        draft_change_list = [
+        # Draft change list with schema version 29.
```

Comment:
Try to make your code as consistent as possible, to improve readability. If this comment is needed here, then it's needed everywhere. Otherwise it's not needed anywhere.

The same applies to e.g. variable naming.